aspnetcore/fundamentals/middleware/index.md

```diff
@@ -171,17 +181,6 @@ The following table shows the requests and responses from `http://localhost:1234
 
 When `Map` is used, the matched path segments are removed from `HttpRequest.Path` and appended to `HttpRequest.PathBase` for each request.
 
-<xref:Microsoft.AspNetCore.Builder.MapWhenExtensions.MapWhen*> branches the request pipeline based on the result of the given predicate. Any predicate of type `Func<HttpContext, bool>` can be used to map requests to a new branch of the pipeline. In the following example, a predicate is used to detect the presence of a query string variable `branch`:
-
-[!code-csharp[](index/snapshot/Chain/StartupMapWhen.cs)]
-
-The following table shows the requests and responses from `http://localhost:1234` using the previous code.
-
-| Request                       | Response                     |
-| ----------------------------- | ---------------------------- |
-| localhost:1234                | Hello from non-Map delegate. |
-| localhost:1234/?branch=master | Branch used = master         |
-
```

Comment:
I moved this down. There was a bit of jumping between `Map`, `MapWhen`, and then `Map` again.